internal/services/devtestlabs/dev_test_virtual_network_resource.go

```diff
@@ -305,18 +336,53 @@ func expandDevTestVirtualNetworkSubnets(input []interface{}, subscriptionId, res
 
 	for _, val := range input {
 		v := val.(map[string]interface{})
+		sharedPublicIpAddressConfiguration := expandDevTestVirtualNetworkSubnetIpAddressConfiguration(v["shared_public_ip_address"].([]interface{}))
```

Comment:
There isn't a need to declare this as a separate variable.
```suggestion
```

Reply:
Will change.